src/main/java/com/sleekbyte/tailor/Tailor.java

```diff
@@ -79,24 +85,45 @@ public static void checkSrcRoot() {
      * @throws IOException if path specified does not exist
      */
     public static Set<String> getSwiftSourceFiles() throws IOException {
-        Set<String> filenames = new TreeSet<>();
+        Set<String> includedItems = new HashSet<>();
+        Set<String> excludedItems = new HashSet<>();
+        Set<String> fileNames = new TreeSet<>();
+
+        // Process config file if present
+        // If config file passed via CLI
+        String configFilePath = argumentParser.getConfigFilePath();
```

Comment:
I think we could move the retrieving of the config path into ConfigurationParser